lib/nexmo_developer/app/controllers/static_controller.rb

```diff
@@ -125,10 +125,6 @@ def podcast
     render layout: 'landing'
   end
 
-  def developer_spotlight
-    render layout: 'landing'
-  end
-
   def migrate
```

Comment:
Should we also remove the `#migrate` action from the controller? It's a landing page in `nexmo-developer` already.

Reply:
done